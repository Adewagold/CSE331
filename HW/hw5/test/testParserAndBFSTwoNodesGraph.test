# test parser and BFS on two nodes graph

# build graph
LoadGraph twoNodesGraph twoNodesGraph.tsv

# check graph is built correctly
ListNodes twoNodesGraph
ListChildren twoNodesGraph node1
ListChildren twoNodesGraph node2

# check BFS
FindPath twoNodesGraph node1 node2