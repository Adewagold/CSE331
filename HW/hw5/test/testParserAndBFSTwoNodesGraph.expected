# test parser and BFS on two nodes graph

# build graph
loaded graph twoNodesGraph

# check graph is built correctly
twoNodesGraph contains: node1 node2
the children of node1 in twoNodesGraph are:
the children of node2 in twoNodesGraph are:

# check BFS
path from node1 to node2:
no path found